### DISCUSSION TITLE
I would like to link to your software on my website

### DISCUSSION THREAD

**Discussion by ajtruckle** [sircharlo/meeting-media-manager#202] [Q&A] [answered]
Dear Brothers

I have a [Other Links](https://www.publictalksoftware.co.uk/other-links/) page on my Theocratic Software website and I would like to add yours to the list.

If you approve, then would you kindly provide:

- Suitable image
- Small paragraph about this software
- Landing URL page

Thank you my brothers.

Andrew

**Comment by sircharlo** [comment, marked as answer]
Thank you! I certainly approve. BTW, I've used your software in the past, and very much enjoyed it. Thanks for your hard work on it, it's a huge project!

I'll write a small paragraph tonight or tomorrow and post it here.

**Reply by sircharlo** [reply]
JW Meeting Media Fetcher is an app for Windows, macOS and Linux, that automatically downloads pictures and videos that will be used at the meetings, in any language found on JW.org. It features recurring and custom media management, congregation sync, and automated MP4 conversion for easy Zoom sharing.

**Reply by sircharlo** [reply]
Icon: 
![](https://github.com/sircharlo/jw-meeting-media-fetcher/raw/master/build/icon.png)

**Reply by sircharlo** [reply]
Landing page: https://sircharlo.github.io/jw-meeting-media-fetcher/

**Reply by sircharlo** [reply]
Let me know if that's what you had in mind @ajtruckle

**Reply by ajtruckle** [reply]
Thank you. When I get time later today I will update my website and report back to you for you to review.

I appreciate your kind words too.

**Reply by ajtruckle** [reply]
I have just updated my links page. Please review. Thank you.

**Reply by sircharlo** [reply]
Thank you very much! Looking good.
